### ISSUE TITLE
Deleting Documents

### ISSUE THREAD

**Issue by rosekal** [jatinrampal/WhiteBears-Capstone#62]
While the remove button per document is fine, there should be a functionality to have checkboxes (or even a select all) to select multiple documents to be deleted.

![image](https://user-images.githubusercontent.com/19345260/48521541-e870eb00-e842-11e8-9f6c-8c402f0ab71c.png)

**Comment by jatinrampal** [comment]
Isn't feasible since the entries on the database would've to be individually updated/deleted.

**Comment by rosekal** [comment]
This is feasible, just loop through the rows and delete them one by one, not in a batch.

**Comment by rosekal** [comment]
Stop closing issues without a valid reason.

**Comment by jatinrampal** [comment]
What you don't understand is that there are other things that need to be solved/done before implementing something that you 'feel' like adding. Was this discussed with others? no. Was anyone else taken into consideration as to what they think about it? no.

**Comment by rosekal** [comment]
"It's low priority, sure, but don't close it for lack of a good reason."  Please read before replying.

We all agreed that assigning low priority issues means that it's not a critical task.  I thought this goes without saying, but this issue doesn't NEED to be fixed, but it's something that THE JUDGES MAY BRING UP.  I know this, you know this, everybody knows this.  You're preaching to the choir when you bring up the fact that high priority tasks need to be completed before issues like this.

You spent months on this one page, this should have been included but ultimately not.  For example, it's very tiresome to delete one document at a time if you look at this project from the client's side. Furthermore, this is your page, not mine.  Don't assign issues to me if it's outside of my scope unless you get consent from yours truly.  It's unprofessional.

**Comment by jatinrampal** [comment]
Exactly. 'This should be included'. This was never brought up in any of the conversations before. Unprofessional is making issues that were never discussed and are improvements rather than bugs.

**Comment by rosekal** [comment]
Talk about missing the point.  Read my comment again, please.  Specifically the first paragraph.

I'm not going to ask for Rafael's or Kishan's approval before creating new issues that only you can do.    Nothing unprofessional about that.  If you feel that the issue isn't feasible, go on and explain yourself.

**Comment by rosekal** [comment]
You're being very immature about this whole situation.  Don't want to do it since it's a little too challenging?  Fine.  Answer to the judges when the time comes, I won't support you.
